src/frontend/parser.mly

```diff
@@ -1055,6 +1061,31 @@ pattern_non_var_bot:
         | utpat2 :: utpat_rest ->
             make_standard (Tok tokL) (Tok tokR) (UTPTuple(TupleList.make utpat1 utpat2 utpat_rest))
       }
+  | tokL=L_RECORD; body=pat_record_body; tokR=R_RECORD
+      {
+        let (is_open, fields) = body in
+        make_standard (Tok tokL) (Tok tokR) (UTPRecord(is_open, fields))
+      }
+;
+pat_record_body:
+  | WILDCARD; COMMA?
```

Comment:
The description `COMMA?` allows `(| foo = x, _, |)` for a record pattern as well as `(| foo = x, _ |)`. Is this intended?

Reply:
Yes, this is intentional. I think there is no reason to forbid a trailing comma after a wildcard.

Reply:
Same for list `[ value1; ]` python also accept trailing comma: one advantage is we can swap item and it will just works. Other reason seems to be human better deal with terminator than separator (last argument was given by an ocp-indent author).